src/contacts.cs

```diff
@@ -27,12 +27,7 @@ interface CNKeyDescriptor : NSObjectProtocol, NSSecureCoding, NSCopying {
 
 	[iOS (9,0), Mac (10,11, onlyOn64: true)]
 	[BaseType (typeof (NSObject))]
-	interface CNContact : NSCopying, NSMutableCopying, NSSecureCoding, NSItemProviderReading
-#if IOS_NOT_BOUND_YET
-	// https://bugzilla.xamarin.com/show_bug.cgi?id=58203
-	, NSItemProviderWriting
-#endif
-	{
+	interface CNContact : NSCopying, NSMutableCopying, NSSecureCoding, NSItemProviderReading, NSItemProviderWriting {
```

Comment:
@spouliot do you still want me to add this info in the commit message? I can add it when merging so I do not issue a new rebuild